static/app/views/performance/transactionSummary/transactionOverview/trendChart/index.tsx

```diff
@@ -116,31 +135,124 @@ function TrendChart({
 
   const trendDisplay = generateTrendFunctionAsString(trendFunction, trendParameter);
 
+  function getSelectedTransaction(
+    trendChangeType: TrendChangeType,
+    transactions?: NormalizedTrendsTransaction[]
+  ): NormalizedTrendsTransaction | undefined {
+    const queryKey = getSelectedQueryKey(trendChangeType);
+    const selectedTransactionName = decodeScalar(location.query[queryKey]);
+
+    if (!transactions) {
+      return undefined;
+    }
+
+    const selectedTransaction = transactions.find(
+      transaction =>
+        `${transaction.transaction}-${transaction.project}` === selectedTransactionName
+    );
+
+    if (selectedTransaction) {
+      return selectedTransaction;
+    }
+
+    return transactions.length > 0 ? transactions[0] : undefined;
+  }
+
+  const trendView = eventView.clone() as TrendView;
+  modifyTrendView(
+    trendView,
+    location,
+    TrendChangeType.REGRESSION,
+    projects,
+    organization as Organization
```

Comment:
Rather than asserting that `organization as Organization`, I think we should question if the `organization` parameter in `modifyTrendView` needs to be an `Organization` at all

`OrganizationSummary` is a subset of fields from `Organization`. We do that so we don't need to request a huge amount of detailed data in our list endpoint. With our types, `Organization extends OrganizationSummary`, so I think we _can_ change the parameter to `OrganizationSummary` and as long as the function doesn't complain, we should be good with using a smaller subset of data.